test/e2e/page-objects/pages/header-navbar.ts

```diff
@@ -1,29 +1,43 @@
 import { Driver } from '../../webdriver/driver';
+import AccountOptionsMenu from './account-options-menu';
 
 class HeaderNavbar {
   private driver: Driver;
 
   private accountMenuButton: string;
 
-  private accountOptionMenu: string;
+  private accountOptionsMenuButton: string;
 
   private lockMetaMaskButton: string;
 
+  private accountOptionsMenu: AccountOptionsMenu;
+
   constructor(driver: Driver) {
     this.driver = driver;
-    this.accountMenuButton = '[data-testid="account-menu-icon"]';
-    this.accountOptionMenu = '[data-testid="account-options-menu-button"]';
     this.lockMetaMaskButton = '[data-testid="global-menu-lock"]';
+    this.accountMenuButton = '[data-testid="account-menu-icon"]';
+    this.accountOptionsMenuButton =
+      '[data-testid="account-options-menu-button"]';
+    this.accountOptionsMenu = new AccountOptionsMenu(driver);
   }
 
   async openAccountMenu(): Promise<void> {
     await this.driver.clickElement(this.accountMenuButton);
   }
 
+  async openAccountOptionsMenu(): Promise<void> {
+    await this.driver.clickElement(this.accountOptionsMenuButton);
+  }
+
   async lockMetaMask(): Promise<void> {
-    await this.driver.clickElement(this.accountOptionMenu);
+    await this.openAccountOptionsMenu();
     await this.driver.clickElement(this.lockMetaMaskButton);
   }
+
+  async goToNotifiationsList(): Promise<void> {
+    this.openAccountOptionsMenu();
+    this.accountOptionsMenu.clickNotificationsMenuItem();
+  }
```

Comment:
With this implementation, for notifications, we use `clickNotificationsMenuItem` in the `AccountOptionsMenu` page, but we click `lockMetaMaskButton` using the selectors in the `header-navbar` page directly. However, the `notificationsMenuItem` and the `lockMetaMaskButton` are on the same modal, which creates inconsistency. I suggest that we don't create a new page class for `AccountOptionsMenu` because it only has a few buttons without complex interactions on `AccountOptionsMenu`. We can just add the `notificationsMenuItem` selector in the ` header-navbar` page and click on it directly here. This provides an easier implementation.
![Screenshot 2024-09-20 at 09 35 19](https://github.com/user-attachments/assets/431982b9-eb67-44fc-9230-a0bacb39f14c)